scripts/zkai_minimal_transformer_block_benchmark_gate.py

```
@@ -0,0 +1,666 @@
+    expected = _base_payload()
+    for key in BASELINE_KEYS:
+        if payload[key] != expected[key]:
+            raise MinimalBlockBenchmarkError(f"{key} drift")
+    rows = payload["component_rows"]
+    if not isinstance(rows, list) or len(rows) != 10:
+        raise MinimalBlockBenchmarkError("component row inventory drift")
+    if any(row.get("comparability") == "MATCHED_EXTERNAL_BENCHMARK" for row in rows if isinstance(row, dict)):
+        raise MinimalBlockBenchmarkError("external comparability overclaim")
+    native_row = next(row for row in rows if row["component"] == "native_full_block_proof_object")
+    if native_row["object_class"] != "missing_native_proof_object" or native_row["primary_value"] is not None:
+        raise MinimalBlockBenchmarkError("native block proof object overclaim")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)